react/Icon/Readme.md

```diff
@@ -580,6 +580,7 @@ import BottomSelectIcon from 'cozy-ui/transpiled/react/Icons/BottomSelect'
 import CheckWhiteIcon from 'cozy-ui/transpiled/react/Icons/CheckWhite'
 import DashWhiteIcon from 'cozy-ui/transpiled/react/Icons/DashWhite'
 import KeychainIcon from 'cozy-ui/transpiled/react/Icons/Keychain'
+import CozyUpgrade2Icon from 'cozy-ui/transpiled/react/Icons/CozyUpgrade2'
```

Comment:
pourquoi Upgrade2 ? Il ne semble pas déjà  y avoir de Upgrade cf https://cballevre.github.io/cozy-ui/react/#!/Icon

Je pense qu'on devrait le nommer plutôt en rapport à l'image, plutôt que le contexte d'utilisation, genre CozyLock ou CozyLockAndSpark ou un truc du genre... ?

Reply:
I follow the name of the illustration into figma. Maybe I can rename the icon. There is an icon named lock on figma so I will synchronize with @joel-costa before to maintain consistency.